### PR TITLE
Add update subscriber list queue consumer restart scripts

### PR DESCRIPTION
**Pull request by huwd**
[Trello](https://trello.com/c/IkYegsOt/1225-update-subscriberlist-titles-on-major-minor-publishing-change)

Dependent on
============
https://github.com/alphagov/email-alert-service/pull/493

See also
========
https://github.com/alphagov/govuk-app-deployment/pull/436/commits/5c4c0e28ac2132bcbafeaab6a75bbfa3a8a11c47

### CODE REVIEW

**Review by alex9smith — DISMISSED** [review]
LGTM

File: email-alert-service/config/deploy.rb [hunk 1]
```diff
@@ -17,7 +17,21 @@
       run "sudo initctl restart email-alert-service-unpublishing-queue-consumer-procfile-worker || "\
           "sudo initctl start email-alert-service-unpublishing-queue-consumer-procfile-worker"
     end
+
+    desc "Restart the update subscriber list details minor consumer"
+    task :restart_subscriber_list_details_update_minor_consumer do
+      run "sudo initctl restart subscriber-list-details-update-minor-consumer-procfile-worker || "\
+          "sudo initctl start subscriber-list-details-update-minor-consumer-procfile-worker"
```
Comment (barrucadu) [inline email-alert-service/config/deploy.rb:24]: These service names (and below) have to match the puppet worker names (not the `process_type`s).  So they'll be `email-alert-service-subscriber-list-details-update-major-consumer` and `email-alert-service-subscriber-list-details-update-major-consumer`

Comment (huwd) [reply email-alert-service/config/deploy.rb:24]: the `email-alert-service` bit makes sense to be, but why no `procfile-worker` on the end?
How does this one differ from the above?

https://github.com/alphagov/govuk-puppet/blob/main/modules/govuk/manifests/procfile/worker.pp#L63

Comment (barrucadu) [reply email-alert-service/config/deploy.rb]: Oh right yes, it does also need the `-procfile-worker` bit too

**Review by barrucadu — APPROVED** [review]

**Pull request merged**